### PR TITLE
Oppenheimer is no longer shot by the syndicate turrets

### PR DESCRIPTION
**Pull request by Therealdoooc213**
## About The Pull Request

In the title

## Why It's Good For The Game

Bugs are bad

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

image

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
fix: Oppenheimer is no longer shot by their own turrets
/:cl:

### CODE REVIEW

**Review by Dejaku51 — CHANGES_REQUESTED** [review]
change faction in the mob define instead of editting it in map. also don't add "syndicate" as a faction. use FACTION_SYNDICATE define

File: code/game/machinery/porta_turret/portable_turret.dm [hunk 1]
```diff
@@ -706,7 +706,7 @@ DEFINE_BUFFER_HANDLER(/obj/machinery/porta_turret)
 	stun_projectile_sound = 'sound/weapons/gunshot.ogg'
 	icon_state = "syndie_off"
 	base_icon_state = "syndie"
-	faction = list(FACTION_SYNDICATE)
+	faction = list(FACTION_SYNDICATE, "syndicate")
```
Comment (Dejaku51) [inline code/game/machinery/porta_turret/portable_turret.dm]: remove the added "syndicate" faction

Comment (Therealdoooc213) [reply code/game/machinery/porta_turret/portable_turret.dm]: Woops

**Comment by Tsar-Salat** [comment]
> change faction in the mob define instead of editting it in map. also don't add "syndicate" as a faction. use FACTION_SYNDICATE define

I don't think we should be mandating this unless someone goes through the effort of converting *all* the faction strings to defines.

It doesn't really matter though

**Review by Tsar-Salat — APPROVED** [review]
Simple change, Dejaku already addressed the mapedit issue. 

Looks good to me

**Review by Dejaku51 — APPROVED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**